src/app/frontend/deploy/deploy.controller.js

```diff
@@ -131,4 +143,24 @@ export default class DeployController {
   isPortMappingEmpty_(portMapping) {
     return !!portMapping.port && !!portMapping.targetPort;
   }
+
+  /**
+   * Initializes namespace selector.
+   * @param {!angular.$log} $log
+   * @param {!angular.$resource} $resource
+   * @private
+   */
+  initializeNamespaces_($log, $resource) {
```

Comment:
Hmm... $resource and $log are already saved as private fields. I suggest using it when they are already available. With such approach we'll have consistent style across the file, which is very nice.